src/backend/distributed/shared_library_init.c

```diff
@@ -689,6 +689,18 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_manual_changes_to_shards",
+		gettext_noop("Disables dropping and truncating known shards."),
+		gettext_noop("Set to true by default. If set to false, enables "
+					 "dropping and truncating shard on the coordinator "
+					 "(or the MX workers)"),
```

Comment:
MX workers -> workers with metadata

(we don't really use MX in user-facing terminology so far)